### ISSUE TITLE
Unable to access `kube_config` in Terraform 0.12

### ISSUE THREAD

**Issue by nick-woodward** [terraform-providers/terraform-provider-digitalocean#239]
### Terraform Version
```sh
$ terraform -v
xxx
```

### Affected Resource(s)

- digitalocean_kubernetes_cluster

### Terraform Configuration Files
```hcl
provider "digitalocean" {}

resource "digitalocean_kubernetes_cluster" "default" {
  name    = "delete-me"
  region  = "nyc1"
  version = "1.14.2-do.0"

  node_pool {
    name       = "worker-pool"
    size       = "s-1vcpu-2gb"
    node_count = 3
  }
}

output "kube_config" {
  value = digitalocean_kubernetes_cluster.default.kube_config[0].host
  # I've also attempted it with the following syntaxes just incase..
  # value = "${digitalocean_kubernetes_cluster.default.kube_config.0.host}"
  # value = digitalocean_kubernetes_cluster.default.kube_config.0.host
  # value = element(digitalocean_kubernetes_cluster.default.kube_config, 0)

  # Accessing the overall set works fine.
  # value = digitalocean_kubernetes_cluster.default.kube_config
}
```

### Debug Output

https://gist.github.com/nick-woodward/f152e6b934a4433d91ef037ea65680e9

### Expected Behavior

Terraform should have allowed me to access the items in the `kube_config`.

### Actual Behavior

Terraform throws an error indicating it believes the attribute I'm looking at isn't a set / list.

### Steps to Reproduce

1. `terraform apply`

**Comment by cmndrsp0ck** [comment]
LGTM!

```
....

output "kube_config" {    
  value = digitalocean_kubernetes_cluster.demo_cluster.kube_config[0].raw_config    
}

output "kube_host" {    
  value = digitalocean_kubernetes_cluster.demo_cluster.kube_config[0].host 
}
```

Now able to successfully execute `terraform output kube_config` and `terraform output kube_host`.

**Comment by andrewsomething** [comment]
This fix was released as version 1.4.0. You can upgrade with:

    terraform init -upgrade

Thanks for the report, and please let us know if you're still experiencing any issues.

**Comment by nick-woodward** [comment]
Awesome, thanks for the quick fix @andrewsomething!
